addons/mail/static/src/utils/common/pdf_thumbnail.js

```diff
@@ -0,0 +1,58 @@
+import { loadPDFJSAssets } from "@web/core/utils/pdfjs";
+
+export async function generatePdfThumbnail(
+    pdfUrl,
+    options = { width: 180, height: 110, stripBase64Prefix: false }
```

Comment:
Any way to not duplicate the default size in several places?
Export a variable from somewhere?